IronSoftware.Drawing/IronSoftware.Drawing.Common/KnownColors.cs

```diff
@@ -0,0 +1,436 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace IronSoftware.Drawing
+{
+    internal class KnownColors
+    {
+        static internal uint[] ArgbValues = new uint[] {
```

Comment:
Would `Dictionary <KnownColor, uint>` be more readable and harder to make mistakes?